src/main/java/net/sf/jabref/logic/importer/ImportException.java

```diff
@@ -0,0 +1,17 @@
+package net.sf.jabref.logic.importer;
+
+public class ImportException extends Exception {
```

Comment:
Well, the code would work with JabRefException instead of ImportException, but you would loose the semantics that the exception is really related to the import and not because JabRef runs out of memory (or another general problem). But I have no strong opinion about it (mainly because you are right that we don't have a general strategy regarding exceptions).

ImportException now derives from JabRefException. Is this a compromise you can live with?